requirements/base.txt

```diff
@@ -3,4 +3,4 @@ jsonschema<5,>=3.2  # TODO: evaluate risk of removing jsonschema 3.x support
 typing_extensions>=4.4,<5 # 3.7 doesn't have Literal
 
 # resource validation & schema generation
-pydantic~=1.8
+pydantic>=1.8
```

Comment:
nit: better convey intent:

```suggestion
pydantic>=1.8,<3
```

Reply:
I'm still not sure we should keep supporting versions <2.0

Reply:
Thank you, I applied this first so that it will not be forgotten when a consensus about keeping pydantic 1.x is reached. If there is a different consensus later, I will change it to `pydantic>=2,<3`.